### ISSUE TITLE
Misleading advertising

### ISSUE THREAD

**Issue by hendorog** [apilayer/marketstack#5]
Marketstack do not provide pricing for all of the exchanges.

They have refunded my purchase on this basis, which is great, but they have not fixed the website.

Please remove references to stock exchanges that you don't provide pricing services for marketstack. 

Thanks

**Comment by Fredrik-C** [comment]
@hendorog 
Yeah I noticed that to, but the advertising on the website is still not changed.
There are lot of gaps in the historical data as well, even for data on large exchanges like NYSE.

Have you found any alternative service to use? 
I think it is safe to discuss competitors here since Marketstack seems to ignore any issues reported here on GitHub ;)

**Comment by Fredrik-C** [comment]
I asked for a refund as well, **twice**... but they just ignore my refund request.

**Comment by benjineering** [comment]
@Fredrik-C did you get your money back? I made the mistake of signing up to this dumpster-fire of a service, now I'm getting null close values and identical OHL values from a simple intraday request for a common symbol, and they're not replying to my messages.

I was thinking of going with Polygon but $$$

**Comment by Fredrik-C** [comment]
@benjineering No money back. Not even a feedback/response/reply even though I re-sent my request.
Its a pure SCAM

**Comment by benjineering** [comment]
Yeah, doing a proper search on Google really only reveals positive articles that have clearly been paid for by Marketstack, no actual reviews

**Comment by jdm365** [comment]
I have had similar issues with lots of missing historical intraday data. It is virtually impossible to create a historical dataset of multiple stocks. The consistency in values returned is awful.

Curious though have any of you found a superior api service. Any help would be greatly appreciated.

**Comment by benjineering** [comment]
@jdm365 I'm having more success with Polygon's free plan https://polygon.io/

**Comment by jdm365** [comment]
Good to know. Thanks! It is pretty pricey though so I may start by developing crypto models using the Tiingo API.
